pkg/lvm/lvm_util.go

```diff
@@ -511,9 +606,25 @@ func parseVolumeGroup(m map[string]string) (apis.VolumeGroup, error) {
 		quantity := resource.NewQuantity(sizeBytes, resource.BinarySI)
 		*value = *quantity //
 	}
+
+	vg.Permission = getIntFieldValue("vg_permissions", m["vg_permissions"])
+	vg.AllocationPolicy = getIntFieldValue("vg_allocation_policy", m["vg_allocation_policy"])
+
 	return vg, err
 }
 
+// This function returns the integer equivalent for different string values for the LVM component(vg,lv) field.
+// -1 represents undefined.
+func getIntFieldValue(fieldName, fieldValue string) int {
+	mv := -1
+	for i, v := range Enums[fieldName] {
+		if v == fieldValue {
+			mv = i
```

Comment:
'break' here or its not available?

Reply:
Yes, break can be used. If none of the values match then the return value will be -1.